package.json

```diff
@@ -4,43 +4,68 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
+  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
   "license": "MIT",
   "scripts": {
-    "build": "npm run lint && tsc",
-    "test": "jest",
+    "build": "tslint --project . && tsc",
+    "clean": "rm -r ./dist",
+    "prettify": "prettier --write './**/*.ts'",
+    "start": "node dist/index.js",
+    "test:unit": "jest --coverage --runInBand",
+    "test:integration": "jest --config=jest.integration.js --coverage --runInBand",
+    "test": "npm-run-all -s test:*",
     "lint": "tslint --project ."
   },
   "devDependencies": {
-    "prettier": "^1.13.5",
+    "@types/bluebird": "^3.5.20",
+    "@types/debug": "0.0.30",
+    "@types/express": "^4.11.1",
+    "@types/jest": "^23.0.0",
+    "@types/morgan": "^1.7.35",
+    "@types/node": "^8.9.4",    
+    "@types/node-fetch": "^1.6.9",
+    "@types/supertest": "^2.0.4",    
+    "@types/uuid": "^3.4.3",
+    "@types/winston": "^2.3.9",
     "danger": "^3.7.16",
-    "tslint": "^5.10.0",
-    "typescript": "^2.9.1",
     "danger-plugin-digitalcitizenship": "^0.3.1",
+    "jest": "^23.1.0",
+    "npm-run-all": "^4.1.3",
+    "prettier": "^1.13.5",
+    "supertest": "^3.0.0",
+    "ts-jest": "^22.4.6",
+    "tslint": "^5.10.0",
     "tslint-config-prettier": "^1.13.0",
     "tslint-immutable": "^4.5.4",
     "tslint-plugin-prettier": "^1.3.0",
     "tslint-sonarts": "^1.7.0",
-    "typestrict": "^0.0.9",
-    "italia-pagopa-api":
-      "https://github.com/teamdigitale/italia-pagopa-api#v0.0.5",
-    "jest": "^23.1.0",
-    "ts-jest": "^22.4.6",
-    "@types/jest": "^23.0.0",
-    "@types/node": "^8.9.4"
+    "typescript": "^2.9.1",
+    "typestrict": "^0.0.9"
+  },
+  "dependencies": {
+    "debug": "^3.1.0",
+    "express": "^4.16.3",
+    "fp-ts": "^1.3.0",
+    "io-ts": "^1.0.5",
+    "italia-pagopa-api": "teamdigitale/italia-pagopa-api#158031250-Manual-Fix-wsdl-to-ts-to-correct-type-recognition",
+    "italia-ts-commons": "^2.1.0",
+    "morgan": "^1.9.0",
+    "node-fetch": "^2.1.2",
+    "querystring": "^0.2.0",
+    "uuid": "^3.2.1",
+    "winston": "^2.4.2"
   },
-  "dependencies": {},
   "jest": {
-    "globals": {
-      "ts-jest": {
-        "skipBabel": true
-      }
-    },
+    "verbose": true,
     "collectCoverage": true,
     "transform": {
-      "^.+\\.tsx?$": "ts-jest"
+      "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
-    "moduleFileExtensions": ["ts", "tsx", "js", "jsx", "json", "node"]
+    "testMatch": [
+      "**/__tests__/*.js"
```

Comment:
this should match `"**/__tests__/*.test.ts"`

Reply:
also, `PaymentConverter.tests.ts` should be renamed to `PaymentConverter.test.ts` - you should use the same jest config from `italia-backend`